src/blackboxTests/src/test/kotlin/org/vaccineimpact/api/blackboxTests/helpers/FluentValidation.kt

```diff
@@ -159,8 +159,8 @@ class FluentValidation(config: FluentValidationConfig)
 
     private fun makeRequest(contentType: String, token: TokenLiteral? = null): Response = when (method)
     {
-        HttpMethod.get -> requestHelper.get(url, token = token, contentType = contentType)
-        HttpMethod.post -> requestHelper.post(url, postData, token = token)
+        HttpMethod.get -> requestHelper.get(url, token = token, acceptsContentType = contentType)
+        HttpMethod.post -> requestHelper.post(url, postData, token = token, acceptsContentType = contentType)
```

Comment:
Sort of unrelated to this change, I renamed this variable throughout the blackbox tests to `acceptsContentType` to be clearer.